doc/whats_new.rst

```diff
@@ -89,6 +89,7 @@ Changelog
 
 Bug
 ~~~
+- Fix :func:`mne.io.BaseRaw.plot_proj_topomap` by `Joan Massich`_
```

Comment:
We typically link as `mne.io.Raw.plot_proj_topomap` since this is mostly where users will see it